content/en/contribute/code/core/automated-tests.md

```diff
@@ -137,6 +137,10 @@ flowchart LR
     e2e-tests o--Pouch/HTTPS--o cht-e2e
 ```
 
+### Tips to write automated e2e tests
+
+Please read the [style guide for automated tests]({{< relref "contribute/code/core/style-guide-automated-e2e-tests.md" >}}) which provides editorial guidelines for anyone creating new automated test cases for CHT-Core.
```

Comment:
Same here, I am not sure if I linked the file correctly

Reply:
Linked correctly 👌🏽.